src/content/app/genome-browser/components/browser-track-config/useBrowserTrackConfig.ts

```diff
@@ -185,35 +185,35 @@ const useBrowserTrackConfig = () => {
     });
   };
 
-  const updateShowTranscriptIds = (isTranscriptIdsShown: boolean) => {
+  const updateShowTranscriptLabels = (isTranscripLabelsShown: boolean) => {
```

Comment:
`is` -> `are` in the function parameter? or perhaps just `shouldShow`, without the `transcriptLabels` part?

Reply:
I have used `shouldShowTranscriptLabels`, Is that OK?

Reply:
Verbose, but ok 🤷‍♂️